recipes/libmaus2/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "libmaus2" %}
+{% set version = "2.0.436_release_20180104101628" %}
+{% set hash = "4d5c0d69091d67d4cfa92b832398974d6490db6b043b47e8e541508cc6a2c9c9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gt1/libmaus2/archive/2.0.436-release-20180104101628.tar.gz
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - snappy
+        - gcc   # [not osx]
+        - llvm  # [osx]
+        - boost {{CONDA_BOOST}}*
+        - libtool
+        - autoconf
+        - automake
+        - pkg-config  
+        - zlib {{ CONDA_ZLIB }}*
+
+    run:
+        - snappy
+        - pkg-config    
+        - libgcc    # [not osx]
+        - zlib {{ CONDA_ZLIB }}*
+
+test:
+  commands:
+    - echo
```

Comment:
Something missing here? If there are no tests I think the whole section can be removed.

Reply:
I tried without a test but lint complained.

Reply:
@mictadlo a good solution here would be to generate and run a minimal compilation example inside a `run_test.sh` file. This will verify the libs are successfully installed and available.

Reply:
@daler Could you please provide an example?

Reply:
Sure, have a look at https://github.com/bioconda/bioconda-recipes/blob/master/recipes/talloc/run_test.sh

Reply:
I noticed that inside `https://github.com/gt1/libmaus2/tree/master/test` are test scripts. Shoud I add those scripts in the following way to meta.yaml?
```
test:
  commands:
    - test/testAIO.sh
    - testBitIO.sh
    
```